runtime/mainnet/src/config/monetary.rs

```diff
@@ -0,0 +1,721 @@
+	type LengthToFee = ConstantMultiplier<Balance, TransactionByteFee>;
+	type OnChargeTransaction = OnChargeTransaction;
+	type OperationalFeeMultiplier = ConstU8<5>;
+	type RuntimeEvent = RuntimeEvent;
+	type WeightInfo = pallet_transaction_payment::weights::SubstrateWeight<Runtime>;
+	type WeightToFee = fee::WeightToFee;
+}
+
+pub(crate) type TreasuryPaymaster<T> = PayFromAccount<T, TreasuryAccount>;
+parameter_types! {
+	pub const SpendPeriod: BlockNumber = 6 * DAYS;
+	pub const TreasuryPalletId: PalletId = TREASURY_PALLET_ID;
+	pub const MaxApprovals: u32 = 100;
+	pub const PayoutPeriod: BlockNumber = 30 * DAYS;
+}
+
+#[cfg(feature = "runtime-benchmarks")]
+pub struct BenchmarkHelper;
+#[cfg(feature = "runtime-benchmarks")]
+impl pallet_treasury::ArgumentsFactory<(), AccountId> for BenchmarkHelper {
+	fn create_asset_kind(_seed: u32) -> () {
+		()
+	}
+
+	fn create_beneficiary(seed: [u8; 32]) -> AccountId {
+		let account_id = AccountId::from(seed);
+		Balances::force_set_balance(
+			crate::RuntimeOrigin::root(),
+			account_id.clone().into(),
+			EXISTENTIAL_DEPOSIT,
+		)
+		.unwrap();
+		account_id
+	}
+}
+impl pallet_treasury::Config for Runtime {
+	type AssetKind = ();
+	type BalanceConverter = UnityAssetBalanceConversion;
+	#[cfg(feature = "runtime-benchmarks")]
+	type BenchmarkHelper = BenchmarkHelper;
+	type Beneficiary = AccountId;
+	type BeneficiaryLookup = IdentityLookup<Self::Beneficiary>;
+	type BlockNumberProvider = System;
+	type Burn = ();
+	type BurnDestination = ();
+	type Currency = Balances;
+	type MaxApprovals = MaxApprovals;
+	type PalletId = TreasuryPalletId;
+	type Paymaster = TreasuryPaymaster<Self::Currency>;
+	type PayoutPeriod = PayoutPeriod;
+	type RejectOrigin = EnsureRoot<AccountId>;
+	type RuntimeEvent = RuntimeEvent;
+	type SpendFunds = ();
+	/// Never allow origins except via the proposals process.
+	type SpendOrigin = NeverEnsureOrigin<Balance>;
+	type SpendPeriod = SpendPeriod;
+	type WeightInfo = pallet_treasury::weights::SubstrateWeight<Runtime>;
+}
+
+#[cfg(test)]
+mod tests {
+	use std::any::TypeId;
+
+	use codec::Encode;
+	use frame_support::{
+		assert_ok,
+		dispatch::GetDispatchInfo,
+		traits::{fungible::Mutate, Get},
+		weights::{constants::ExtrinsicBaseWeight, Weight, WeightToFee},
+	};
+	use pallet_transaction_payment::OnChargeTransaction as OnChargeTransactionT;
+	use pop_runtime_common::{MICRO_UNIT, MILLI_UNIT};
+	use sp_keyring::sr25519::Keyring;
+	use sp_runtime::{traits::Dispatchable, BuildStorage};
+
+	use super::*;
+	use crate::{AccountId, RuntimeCall, RuntimeOrigin, UNIT};
+
+	type OnChargeTransaction = <Runtime as pallet_transaction_payment::Config>::OnChargeTransaction;
+
+	// fn new_test_ext() -> sp_io::TestExternalities {
+	// 	let storage = frame_system::GenesisConfig::<Runtime>::default().build_storage().unwrap();
+	// 	let mut ext = sp_io::TestExternalities::new(storage);
+	// 	ext.execute_with(|| System::set_block_number(1));
+	// 	ext
+	// }
```

Comment:
commented code should be removed

Reply:
[6ff08a0](https://github.com/r0gue-io/pop-node/pull/458/commits/6ff08a05e4f7bf4cdf98f8aa1a5db3420ed10d8d)